api-specificatie/openapi.yaml

```diff
@@ -1,65 +1,59 @@
 openapi: 3.0.0
 servers:
+# Added by API Auto Mocking Plugin
   - description: "SwaggerHub API Auto Mocking"
     url: https://virtserver.swaggerhub.com/VNGRealisatie/api/brp_update_api
 info:
   title: "BRP Update API"
-  description: |
-        Het instellen van volgindicaties op ingeschrevenpersonen, het beëindigen van die volgindicaties, het opvragen van alle van voor een abonnee ingestelde volgindicaties en het opvragen van de aanwezigheid van wijzigingen op ingeschrevenpersonen.
-        Een volgindicatie kan worden beëindigd door het opgeven van een deleteDatum.
+  description: "Deze API biedt functionaliteit voor het opvragen van  wijzigingen op ingeschreven personen die door de gebruiker worden gevolgd vanaf een bepaald moment. Dit kan gebruikt worden om een kopie aan te leggen van de actuele registratie van specifieke personen.<br/>
```

Comment:
Met deze API kun je opvragen welke door jou gevolgde personen zijn gewijzigd in de BRP. Je kunt je abonneren op een persoon die je wilt volgen, en je kunt opvragen welke personen door jou worden gevolgd. Je kunt deze API gebruiken in combinatie met de BRP bevragen API, bijvoorbeeld om lokale kopiegegevens actueel te houden.